index.html

```diff
@@ -36,7 +36,158 @@ <h2>
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </div>
+                <button>See Project</button>
+            </div>
+            <div class="cards">
+                <h1>
+                    Profesional Art Printing Data
+                </h1>
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </div>
+                <button>See Project</button>
+            </div>
+            <div class="cards">
+                <h1>
+                    Profesional Art Printing Data
+                </h1>
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </div>
+                <button>See Project</button>
+            </div>
+        </section>
+        <section class="About">
+            <div>
+                <h1>
+                    About me
+                </h1>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <button>
+                    Get My Resume
+                </button>
+            </div>
+            <div >
+                <div class="properties">
+                    <h2>Language</h2>
+                    <h3>Javascript</h3>
+                    <h3>Ruby</h3>
+                    <h3>Html</h3>
+                    <h3>Css</h3>
+                </div>
+                <div class="properties">
+                    <h2>Frameworks</h2>
+                    <h3>Bootstrap</h3>
+                    <h3>Ruby on Rails</h3>
+                    <h3>RSPec</h3>
+                    <h3>CapyBara</h3>
+                    <h3>Selenium</h3>
+                </div>
+                <div class="properties">
+                    <h2>Skills</h2>
+                    <h3>Codekit</h3>
+                    <h3>Github</h3>
+                    <h3>Codepen</h3>
+                    <h3>Gitlab</h3>
+                    <h3>Terminal</h3>
+                </div>
+            </div>
```

Comment:
- [ ] Kindly I recommend to you use a `ul,` each of them in a `li` for a technology categories ("Languages", "Frameworks" and "Skills") and for The technology in each category(e.g., "JavasScript", "HTML" and "CSS") . This way will be semantically correct and the screen readers and other technologies will thank you.